pkg/controller/history/controller_history_test.go

```diff
@@ -258,8 +258,8 @@ func TestRealHistory_CreateControllerRevision(t *testing.T) {
 		history := NewHistory(client, informer.Lister())
 
 		var collisionCount int32
-		for i := range test.existing {
-			_, err := history.CreateControllerRevision(test.existing[i].parent, test.existing[i].revision, &collisionCount)
+		for _, exst := range test.existing {
```

Comment:
`exst` -> `exists`

Reply:
Renamed the iteration variable to` item`. I think exists just sounds too much like a boolean variable and the name is also used by another variable in the function.